common/src/main/java/be/portal/job/mappers/application/ApplicationMapper.java

```diff
@@ -11,19 +11,22 @@
 import org.mapstruct.Mapping;
 import org.mapstruct.MappingTarget;
 
+import java.time.LocalDateTime;
 import java.util.Set;
 import java.util.stream.Collectors;
 
 @Mapper(componentModel = "spring")
 public interface ApplicationMapper {
 
     @Mapping(target = "id", ignore= true)
+    @Mapping(target = "applyDate", ignore = false)
     @Mapping(target = "createdAt", ignore = true)
     @Mapping(target = "updatedAt", ignore = true)
     @Mapping(source = "jobSeeker", target = "jobSeeker")
```

Comment:
Put your mapping of applyDate above jobSeeker mapping, not under the id mapping just to have a better readability